advisories/0012.md

```diff
@@ -0,0 +1,33 @@
+### Mirantis Container Products and CVE-2024-3094: Not Vulnerable
+
+#### Release Date
+2024-04-16
+
+#### Overview
+This advisory addresses the relevance of CVE-2024-3094, a vulnerability identified within the xz package, and its implications for Mirantis Container Products (MCR, MSR, and MKE). While Mirantis has analyzed the impact of this vulnerability on the container products and found that they are not vulnerable, customers should always consult their Operating System vendor to ensure that the remaining portions of their deployment are also secure.
+
+#### Unaffected Products
+None of MCR (Mirantis Container Runtime), MSR (Mirantis Secure Registry), or MKE (Mirantis Kubernetes Engine) are vulnerable.
+
+As with all deployments, the operating system that hosts the usage of Mirantis container products may be vulnerable. Despite the information below, the host system should always be verified with the operating system vendor to ensure that it is not vulnerable. Though no additional risk is introduced through the use of the Mirantis products listed in this bulletin, taking this step will ensure that the complete deployment is secured.
+
+Mirantis Container Runtime (MCR) is shipped as operating system packages. As such, MCR itself does not contain or enable the installation of any affected versions of xz.
```

Comment:
MCR was already mentioned earlier at line 10, shall we just use MCR instead of the full name again here?  Same for the MSR and MKE below.

Reply:
I left the paragraphs starting with full name, because it looks awkward starting with an acronym.